libs/api/domains/email-signup/src/lib/serviceProvider.ts

```diff
@@ -0,0 +1,13 @@
+import { Provider } from '@nestjs/common'
+import { ConfigType, LazyDuringDevScope } from '@island.is/nest/config'
+import { EmailSignupConfig } from './emailSignup.config'
+import { EmailSignupService } from './emailSignup.service'
+
+export const EmailSignupServiceProvider: Provider<EmailSignupService> = {
```

Comment:
We use this "Provider" pattern to wrap classes that have been generated for us, like openapi clients.

In this case, you can mark EmailSignupService directly with `@Injectable()` and  `@Scope(LazyDuringDevScope)` (writing from memory).

Reply:
I think I managed to do it in a recent commit I pushed just now